dlc_developer_config.toml

```diff
@@ -31,14 +31,14 @@ benchmark_mode = false
 [build]
 # Add in frameworks you would like to build. By default, builds are disabled unless you specify building an image.
 # available frameworks - ["autogluon", "huggingface_tensorflow", "huggingface_pytorch", "huggingface_tensorflow_trcomp", "huggingface_pytorch_trcomp", "pytorch_trcomp", "tensorflow", "mxnet", "pytorch"]
-build_frameworks = ["tensorflow"]
+build_frameworks = []
```

Comment:
We need to turn it on manually and enable test to run in separate commit. Before merge the change, we need to turn them back off. Otherwise dlc-pr-quick-checks will fail.

I have several commit to cover different framework version (2.9, 2.10, 2.11) as well as different platform (x86, graviton). In the last commit. turn off all the test for merge.